libs/go/libp2p_node/aea/api.go

```diff
@@ -325,6 +324,7 @@ func (aea *AeaApi) Init() error {
 		aea.pipe = NewPipe(aea.msgin_path, aea.msgout_path)
 	}
 
+	aea.acn_status_chan = make(chan *acn.Status, 1)
```

Comment:
Does implementation ensure there can be at most one status in channel?

Reply:
message processed one by one, so no next mnessage till confirmation receivied

Reply:
Not correct I think, see below.